css-fonts/styles.css

```diff
@@ -0,0 +1,12 @@
+body > h1 {
+  font-family: Poppins;
+}
+
+body h2,
+p {
+  font-family: Playpen Sans;
+}
```

Comment:
We'll discuss in our Q&A, but you need to include a backup font as well

Reply:
Thank you Shawn. I'll remember for next time :)